packages/exceptions/src/exceptions/TransactionException.ts

```diff
@@ -12,10 +12,17 @@ export class TransactionException extends ConcreteException {
   public parse(): void {
     const { message } = this
 
-    const { message: parsedMessage, code } =
-      mapFailedTransactionMessage(message)
+    const {
+      message: parsedMessage,
+      code,
+      module,
+    } = mapFailedTransactionMessage(message)
 
     this.setMessage(parsedMessage)
     this.setContextCode(code)
```

Comment:
In some cases we pass the code within the constructor, like [here](https://github.com/InjectiveLabs/injective-ts/blob/3947185308da9ed553ad6686b281d87fcbeec889/packages/sdk-ts/src/core/modules/tx/api/TxRestApi.ts#L58-L61) so we should just use that instead of using the parsed one